api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -947,6 +981,20 @@ type ProtoPort struct {
 	Net string `json:"net,omitempty"`
 }
 
+// +kubebuilder:validation:Pattern=`^(([0-9]*(\.[0-9]*)?(ms|s|h|m|us)+)+|auto)$`
+type BPFConntrackTimeout string
+
+type BPFConntrackTimeouts struct {
+	CreationGracePeriod *BPFConntrackTimeout `json:"creationGracePeriod,omitempty"`
```

Comment:
```suggestion
        // CreationGracePeriod configures . . . . If set to "Auto" the default value is learned from Linux. If nil, Calico will use its own default value. [Default: 10s]
	CreationGracePeriod *BPFConntrackTimeout `json:"creationGracePeriod,omitempty"`
```

e.g., this field "CreationGracePeriod" isn't fully described in the BPFConntrackTimeouts so it's not obvious what it does from reading this.

Reply:
Actually, CreationGracePeriod doesn't support "auto" does it? i.e., based on the top level comment, there is no linux mapping for this field?